curr_scraper/settings.py

```diff
@@ -69,11 +69,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'DATABASE_URL': os.environ.get('DATABASE_URL'),
     }
 }
-
+ATOMIC_REQUESTS = True 
```

Comment:
Its not gonna affect anything, please check:
https://docs.djangoproject.com/en/4.0/ref/settings/#databases